.github/workflows/linux_sanitizers.yml

```diff
@@ -468,3 +469,86 @@ jobs:
           name: test-results-cpp
           path: ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
           if-no-files-found: 'error'
+  CPU_Functional_Tests:
+    name: CPU functional tests
+    timeout-minutes: 30
+    runs-on: ${{ inputs.runner }}
+    container:
+      image: ${{ inputs.image }}
```

Comment:
Please substitute these `${{ inputs.... }}` with the actual values like in: https://github.com/openvinotoolkit/openvino/pull/25441/commits/06fbf729a5e83332b978445adbb63115c600a5c4#diff-ac3f8d84a8e20a72d9234847a55d421dc4ef7e838fdeee3bb02a77599e9350c0R209-R214